presto-native-execution/presto_cpp/main/common/Configs.h

```diff
@@ -125,6 +130,10 @@ class ConfigBase {
     return optionalProperty(std::string{propertyName});
   }
 
+  /// Returns "N<capacity_unit>" as string containing capacity in bytes.
```

Comment:
I'm not sure how this method works. Does it read "32MB" from file and return it as "33554432"?

Would it make sense to have the logic of parsing "32MB" and converting it to number of bytes in .cpp file as a function in anonymous namespace?

Reply:
@mbasmanova 

> I'm not sure how this method works. Does it read "32MB" from file and return it as "33554432"?
Almost. It reads "32MB" from the property, not file (we already loaded the file), then it calls velox::toCapacity(str, CapacityUnit::BYTE) and then it converts the big number back to string as "33554432".
I believe I had a second line of comment showing example, but seems like got lost on the way to PR.

> Would it make sense to have the logic of parsing "32MB" and converting it to number of bytes in .cpp file as a function in anonymous namespace?
Yes, we can do it too.
Then it will look like
```
      {QueryConfig::kMaxArbitraryBufferSize,
       capacity2String(systemConfig->optionalProperty(
           SystemConfig::kSinkMaxBufferSize).value())
```
instead of how it is now:
```
      {QueryConfig::kMaxArbitraryBufferSize,
       systemConfig->capacityPropertyAsBytesString(
           SystemConfig::kSinkMaxBufferSize)},
```

I don't see much difference in these.